docker/start-puma.sh

```diff
@@ -31,6 +29,6 @@ else
   then
     cp commit_id.txt public/
   fi
-
-  exec /usr/bin/supervisord -c /etc/supervisor/supervisord.conf
 fi
+
+exec bundle exec puma -C config/puma.rb
```

Comment:
will this start logging to the main file logger again? seems it's env var related https://github.com/zooniverse/caesar/blob/896646e0498c8872b08cc1e1d8845e3c8b8e12b4/config/environments/production.rb#L83

Reply:
Nice - it looks like it wasn't ever setup to log to stdout only via supervisor upon more reading.